django_mongodb/expressions.py

```diff
@@ -13,7 +15,22 @@ def value_agg(self, compiler, connection):  # noqa: ARG001
     return {"$literal": self.value}
 
 
+def expression(self, compiler, connection):
+    return self.expression.as_mql(compiler, connection)
+
+
+def expression_agg(self, compiler, connection):
+    return self.expression.as_mql_agg(compiler, connection)
+
+
+def cast(self, compiler, connection):  # noqa: ARG001
+    raise NotSupportedError("Cast is not supported on this database backend")
+
+
 def register_expressions():
     Col.as_mql = col
     Value.as_mql = value
     Value.as_mql_agg = value_agg
+    ExpressionWrapper.as_mql = expression
+    ExpressionWrapper.as_mql_agg = expression_agg
```

Comment:
I think I made a mistake requiring the definition of both `as_mql` and `as_mql_agg` when so often they're the same. See first commit of https://github.com/mongodb-labs/django-mongodb/pull/37 for a proposed solution.